integration_tests/Cargo.toml

```diff
@@ -589,6 +589,10 @@ path = "../rusoto/services/support"
 optional = true
 path = "../rusoto/services/swf"
 
+[dependencies.rusoto_timestream_query]
+optional = true
+path = "../rusoto/services/timestream-query"
```

Comment:
You need to add `timestream-write` here too?

Reply:
Added